tests/test_import.py

```diff
@@ -1,3 +1,5 @@
+from mock import Mock
+
 from tests.tools import AgentTestCase
```

Comment:
nit; Other tests are importing Mock from test.tools. Can you follow that pattern?
`from tests.tools import AgentTestCase, Mock`

Reply:
@vrdmr Thanks for pointing that out, I had not noticed it.

But, should we actually be importing from tests.tools in the other tests? Does tests.tools have its own Mock that provides extra value on top of mock.Mock? If not, I think we should actually be importing from mock in all tests.